config/proto/service.proto

```diff
@@ -0,0 +1,38 @@
+syntax = "proto3";
+
+package hermes;
+
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+import "github.com/googleinterns/step224-2020/config/proto/targets.proto";
+
+option go_package = "github.com/googleinterns/step224-2020/config/proto";
+
+service Hermes {
+
+  // Start monitoring a new storage system.
+  rpc StartMonitoringStorageSystem(HermesProbeRequest) returns (HermesProbeResponse) {}
+
+  // Stop monitoring a storage system that is currently being monitored.
+  rpc StopMonitoringStorageSystem(StopMonitoringSystemRequest) returns (StopMonitoringSystemResponse) {}
+
+  // Lists the storage systems being monitored at the moment.
+  rpc ListMonitoredStorageSystems(ListMonitoredSystemsRequest) returns (ListMonitoredSystemsResponse) {}
+}
+
+message HermesProbeRequest {
+  required HermesProbeDef probe_confg = 1;
+}
+
+message HermesProbeResponse {}
+
+message StopMonitoringSystemRequest {
```

Comment:
```suggestion
// A target must be specified so that Hermes knows which target to stop monitoring.
message StopMonitoringSystemRequest {
```

Reply:
You define target here as a repeated field, so make the comment reflect this by mentioning that at least one targets must be provided. Also what happens if no targets are provided as input? What about many targets? In case of several targets, will they be processed in order?

It's alright not to know now, but just keep in mind that these should probably be updated in the future when you implement these or make use of them in Go.